README.md

```diff
@@ -89,13 +89,7 @@ The component is responsible for managing all the aspects of the performance tes
 This tutorial will guide through Kangal installation process and usage.
 
 ### Installing using helm
-First, let's create the Custom Resource Definition by running:
-
-```shell
-$ kubectl apply -f https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
-```
-
-Add the repository to Helm:
+First, add the repository to Helm:
```

Comment:
does helm install CRDs now? IIRC there were some difficulties with it

Reply:
yes, just leave the CRD inside a folder called `crds` and helm will install if not exists

the difficulties are that helm does not support deletion or update